src/plopp/backends/matplotlib/image.py

```diff
@@ -235,7 +237,7 @@ def format_coord(self, x: float, y: float) -> str:
                     unit=self._hover_slicing['unit'][1],
                 ),
             ]
```

Comment:
I don't know if it is noticeable as lag, but this function does quite a few things (such as allocating new variables, dict and list accesses, some f-string bits) that could be avoided. Have you considered moving those bits to a place where it is done only once? Here you would then only, e.g., set the x and y values in the existing variables, format the remaining bits, and to the index lookup.

Reply:
I tried to improve this slightly.